docs/api/utils/use-table.md

```diff
@@ -255,6 +255,18 @@ Selects all records.
 
 De-selects all records.
 
+### `selectPage`
+
+- Type: `Function`
+
+Selects paged records.
```

Comment:
Please include usage example, because this is very unclear what it does.

Reply:
Most of the time, datasets are presented in pagination,

The purpose of this feature is to provide a function to select all records for the current page

Reply:
Oh okay, the naming was unclear. This is better, thanks!